varnish-macros/src/generator.rs

```diff
@@ -178,6 +178,7 @@ impl Generator {
                     VCL_DURATION,
                     VCL_INT,
                     VCL_IP,
+                    VCL_PROBE,
```

Comment:
Duuuuuuuuee, just landed and implemented that one while flying as `reqwest` needs it.

Welp, the good thing is that I'm pretty close to what you did here \o/

Reply:
LOL, the famous "merge conflict after flying" issues :)

I just hacked it in a few minutes trying to figure out if it will work or not.  I don't think its much of a  waste because you now have far better understanding of how it works :)  The issue though is that `Option` in Rust could mean both `nullable` and `optional` from Varnish perspective - and unless Varnish can ensure values like `SocketAddr` are always non-null, we will have to do some heuristics plus some extra attributes... Not sure though.